Dockerfile

```
@@ -62,11 +62,17 @@ COPY --from=postgresql_container_source /postgresql-container/10/root/usr/libexe
 # This image must forever use UID 26 for postgres user so our volumes are
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
-RUN yum -y --setopt=tsflags=nodocs install \
+RUN if [ $(uname -m) != "s390x" ] ; then yum -y --setopt=tsflags=nodocs install \
       http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-8-2.el8.noarch.rpm \
       http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8-2.el8.noarch.rpm && \
     yum -y module enable postgresql:10 && \
-    INSTALL_PKGS="rsync tar gettext bind-utils nss_wrapper postgresql-server postgresql-contrib" && \
+    yum -y --setopt=tsflags=nodocs install postgresql-server postgresql-contrib && \
+    rpm -V postgresql-server postgresql-contrib ; \
+    else \
+    yum -y install /opt/app-root/src/bin-rpm-dir/postgresql-10.19-1.el8.s390x.rpm  /opt/app-root/src/bin-rpm-dir/postgresql-contrib-10.19-1.el8.s390x.rpm  /opt/app-root/src/bin-rpm-dir/postgresql-server-10.19-1.el8.s390x.rpm && \
```

Hardcoding the version numbers like this is going to be a pain as we try to keep this up to date, because we will continually have to change this line.  If possible, can we do

```suggestion
    yum -y install /opt/app-root/src/bin-rpm-dir/postgresql-*.el8.s390x.rpm  /opt/app-root/src/bin-rpm-dir/postgresql-contrib-*.el8.s390x.rpm  /opt/app-root/src/bin-rpm-dir/postgresql-server-*.el8.s390x.rpm && \
```